app/javascript/authentication/SignInForm.tsx

```diff
@@ -10,40 +10,76 @@ import {
   AlertBox,
   SiteAlert,
   LinkButton,
+  debounce,
 } from "@bloom-housing/ui-components"
-import { useForm } from "react-hook-form"
+import { Dialog, Link, Heading, Alert } from "@bloom-housing/ui-seeds"
+import { FieldValues, SubmitHandler, useForm } from "react-hook-form"
 
 import { getMyAccountPath } from "../util/routeUtil"
-import UserContext from "./context/UserContext"
 import EmailFieldset from "../pages/account/components/EmailFieldset"
 import PasswordFieldset from "../pages/account/components/PasswordFieldset"
-import { Link, Heading } from "@bloom-housing/ui-seeds"
 import "../pages/account/styles/account.scss"
+import { AxiosError } from "axios"
+import { confirmEmail, signIn } from "../api/authApiService"
 
-const SignInForm = () => {
-  const { signIn } = useContext(UserContext)
```

Comment:
This got lost and I think this is what is supposed to be the sign in function, not the import from authApiService